software/otterwiki.yml

```
@@ -0,0 +1,11 @@
+name: "Otter Wiki"
+website_url: "https://github.com/redimp/otterwiki"
+source_code_url: "https://github.com/redimp/otterwiki"
+description: "Simple, easy to use wiki software using markdown."
+licenses:
+  - MIT
+platforms:
+  - Python
+  - Docker
```

```suggestion
platforms:
  - Docker
```

The only supported installation method uses Docker (installation in a python virtualenv is only mentioned as a development setup)